util/Migrator/DbScripts/2022-07-20_00_ProviderOrganizationListDetails.sql

```diff
@@ -0,0 +1,20 @@
+-- Add columns 'UserCount', 'Seats' and 'Plan'
+CREATE OR ALTER VIEW [dbo].[ProviderOrganizationOrganizationDetailsView]
+AS
+SELECT
+    PO.[Id],
+    PO.[ProviderId],
+    PO.[OrganizationId],
+    O.[Name] OrganizationName,
+    PO.[Key],
+    PO.[Settings],
+    PO.[CreationDate],
+    PO.[RevisionDate],
+    (SELECT COUNT(1) FROM [dbo].[OrganizationUser] OU WHERE OU.OrganizationId = PO.OrganizationId AND OU.Status = 2) UserCount,
+    O.Seats Seats,
```

Comment:
I'm nowhere near a SQL expert, just curious on why you're aliasing the seats selection. Is something like `O.[Seats]` not applicable here?

Reply:
It is yes, you're quite right. I will remove the alias.